src/common/readout/vmm3/VMM3Config.cpp

```diff
@@ -43,10 +132,18 @@ void VMM3Config::loadAndApplyCalibration(std::string CalibFile) {
 
   for (auto &Calibration : Calibrations) {
     std::string HybridId = Calibration["VMMHybridCalibration"]["HybridId"];
+    if (!validHybridId(HybridId)){
+      throw std::runtime_error("Invalid HybridID in Calibration file");
+    }
     applyCalibration(HybridId, Calibration);
   }
 }
 
+bool VMM3Config::validHybridId(std::string HybridID){
+  std::regex HybridIdRegex ("E55[0-9]{29}");
```

Comment:
Just check it's a 32 length string
Remove import of regex

Reply:
...